app/client/src/components/pages/Community.Tabs.Monitoring.js

```diff
@@ -1027,7 +1027,11 @@ function MonitoringTab({ monitoringDisplayed, setMonitoringDisplayed }) {
                 <em>{watershed}</em> totals:{' '}
               </strong>
             </p>
-            <table css={modifiedTableStyles} className="table">
+            <table
+              css={modifiedTableStyles}
+              id="monitoring-totals"
```

Comment:
```suggestion
              aria-label="Monitoring Totals"
```

For HMW, we try to avoid selecting based on ids for our Cypress tests. We want to try to use `findByRole` as much as possible for future tests, since it is considered best practice and tests accessibility. Here is the [recommended priority](https://testing-library.com/docs/queries/about/#priority) for using these selectors. Can you make that update? 

Eventually we are going to want to switch to using `findByRole` and `findByText` for our existing tests.  That will likely be a separate PR sometime in the future.